README.md

```diff
@@ -593,6 +593,19 @@ cd <project-root>
 make
 ```
 
+**Build non-amd64 image on amd64:**<br>
```

Comment:
NIT: this just looks a bit clumsy heading :smile: 

How about `Multiarch builds` or `Build for target architectures` or smth? If AMD64 is mentioned I think it should be capital case.